### DISCUSSION TITLE
Control over context

### DISCUSSION THREAD

**Discussion by erickirt** [cline/cline#510] [Feature Requests]
It would be valuable to manage the context of previous conversations and switch models seamlessly during chats. This feature would truly set Cline apart from other solutions. People tend to prefer longer, more continuous discussions, perhaps shifting away from a strictly task-focused approach. As you know, programming is never straightforward—it's more like a rollercoaster, where each peak provides different insights.

**Comment by TomLucidor** [comment]
If only there ware ways to conveniently dump history into chat... or even Git based file changes
